integrations/to.etc.domui.selenium/src/main/java/to/etc/domui/webdriver/core/base/InputPO.java

```diff
@@ -0,0 +1,12 @@
+package to.etc.domui.webdriver.core.base;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openqa.selenium.By;
+import to.etc.domui.webdriver.core.WebDriverConnector;
+
+@NonNullByDefault
+public class InputPO extends AbstractTextPO {
```

Comment:
I noticed we have this one, but not a Text2PO / TextPO... Those should be there (especially for Text2) because their behavior is more complex.